CleverTapSDK/CTUserInfoMigrator.h

```diff
@@ -6,9 +6,10 @@
 //
 
 #import <Foundation/Foundation.h>
+#import "CleverTapInstanceConfig.h"
 
 @interface CTUserInfoMigrator : NSObject
 
-+ (void)migrateUserInfoFileForAccountID:(NSString *)acc_id deviceID:(NSString *)device_id;
++ (void)migrateUserInfoFileForAccountID:(NSString *)acc_id deviceID:(NSString *)device_id config:(CleverTapInstanceConfig *) config;
```

Comment:
This change breaks the `CTUserInfoMigratorTest` tests, please see the other comment about this method parameters and address the tests, as well.